templates/server/puppetserver/conf.d/webserver.conf.erb

```diff
@@ -27,4 +27,19 @@ webserver: {
         <%= cipher_suite %>,
     <%- end -%>
     ]
+<%- if @server_acceptor_threads -%>
```

Comment:
For them to be in the local scope you need to "import" them in the `puppet::server::puppetserver` class. We also "import" them in `puppet::server`.